hepstats/hypotests/calculators/frequentist_calculator.py

```diff
@@ -296,16 +67,16 @@ def qnull(self, poinull, poialt, onesided, onesideddiscovery, qtilde=False):
 
         for p in poinull:
             toysresult = toysresults[p]
-            nll1 = toysresult["nll"][p]
-            nll2 = toysresult["bestfit"]["nll"]
-            bestfit = toysresult["bestfit"]["values"]
+            nll1 = toysresult.nlls[p]
```

Comment:
I wonder if you really need to define these nll1 and nll2, whose names do not speak for themselves immediately. If I'm not mistaken they are only used once, hence the definitions are unnecessary.

Reply:
`q = nll1 - nll2` as explained in `BaseCalculator.q` function, I just use the same definitions.